oap-server/server-library/library-server/pom.xml

```diff
@@ -59,5 +59,21 @@
             <groupId>org.eclipse.jetty</groupId>
             <artifactId>jetty-servlet</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.netty</groupId>
+            <artifactId>netty-handler</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.netty</groupId>
+            <artifactId>netty-codec</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.netty</groupId>
+            <artifactId>netty-codec-http</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.netty</groupId>
+            <artifactId>netty-codec-http2</artifactId>
+        </dependency>
```

Comment:
No code change in this maven module, why change library?

Reply:
@kezhenxu94 reminded me what there is a CVE issue in netty 4.1.45. Therefore we upgraded the netty's bom to 4.1.65 and remove this version.
I excluded netty from grpc-netty, because grpc-netty included netty 4.1.45. So that we have to explicitly add those dependencies. (the same as above)